src/Commands/defaultElevator.cpp

```diff
@@ -0,0 +1,51 @@
+// RobotBuilder Version: 2.0
+//
+// This file was generated by RobotBuilder. It contains sections of
+// code that are automatically generated and assigned by robotbuilder.
+// These sections will be updated in the future when you export to
+// C++ from RobotBuilder. Do not put any code or make any change in
+// the blocks indicating autogenerated code or it will be lost on an
+// update. Deleting the comments indicating the section will prevent
+// it from being updated in the future.
+
+
+#include "defaultElevator.h"
+
+// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTOR
+
+defaultElevator::defaultElevator(): Command() {
+        // Use requires() here to declare subsystem dependencies
+    // eg. requires(chassis);
+    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
+	Requires(Robot::elevator.get());
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
+}
+
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTOR
+
+// Called just before this Command runs the first time
+void defaultElevator::Initialize() {
+
+}
+
+// Called repeatedly when this Command is scheduled to run
+void defaultElevator::Execute() {
+RobotMap::elevatorleftSpool->Set(0.0);
+RobotMap::elevatorrightSpool->Set(0.0);
```

Comment:
Commands should not be using `RobotMap`. Use the subsystem that owns `elevatorLeftSpool`, etc.